### PR TITLE
Fix Qt thread id comparison

### PR DESCRIPTION
**Pull request by anosach-luxoft**
@dev-gh please review and try on your machine

### CODE REVIEW

File: src/components/utils/src/threads/thread_qt.cc [hunk 1]
```diff
@@ -36,6 +36,18 @@
 
 #include <QtConcurrent/QtConcurrent>
 
+namespace {
+
+// AN: Temporary workaround for Windows platform.
+// Thread ID getting should be reworked to be crossplatform
+uint64_t ConvertQtThreadHandleToUInt64(Qt::HANDLE handle) {
+  const DWORD* win_thread_id =
+      reinterpret_cast<DWORD*>(QThread::currentThreadId());
```
Comment (pestOO) [inline src/components/utils/src/threads/thread_qt.cc]: Looks like you ned to convert `handle` instead of `currentThreadId'.

Comment (anosach-luxoft) [inline src/components/utils/src/threads/thread_qt.cc]: @pestOO I don't understand you. `QThread::currentThreadId()` returns pointer to `Qt::HANDLE`

Comment (anosach-luxoft) [inline src/components/utils/src/threads/thread_qt.cc]: @pestOO sorry, my mistake. I'll change it

**Comment by anosach-luxoft** [comment]
@pestOO @dev-gh updated

**Comment by dev-gh** [comment]
@anosach-luxoft 
I've used the same approach here https://github.com/anosach-luxoft/sdl_core/pull/19
Reviewed

**Comment by anosach-luxoft** [comment]
Fix will be provided within #19

**Pull request closed without merging**